packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/use-client-credentials-grant-flow/index.md

```diff
@@ -0,0 +1,116 @@
+
+You can use a tool such as this [JSON Web Key Generator](https://mkjwk.org/) to generate an RSA 2048 JWKS key pair for testing. 
+
+> Note: Use the JSON Web Key Generator link to generate JWKS key pairs for testing purposes only. For a production use case, use your own [internal instance](https://github.com/mitreid-connect/mkjwk.org) of the key pair generator.
+
+The JWKS should look something like this:
+
+```json
+{
+    "keys": [
+        {
+             "kty": "RSA",
+            "e":"AQAB",
+            "kid": "jwks1",
+            "n":"AJncrKuine49_CE….RL0HU=" // key truncated for brevity
+        }    
+    ]
+}
+```
+## Create a Service app
+Create an OAuth Service client app and register the public key with the Service app using the `/apps` API. The following example uses the **Add OAuth2 Client App** request in Postman:
+
+1. In Postman, select the **Add OAuth2 Client App** request within the **Apps** collection.
+2. On the **Body** tab, make the following noted changes to the parameter values and add the JWKS section.
+
+    For example:
+    ```json
+    {
+        "name": "oidc_client",
+        "label": "Name your app", //enter a name for the Service app
+        "signOnMode": "OPENID_CONNECT",
+        "credentials": {
+            "oauthClient": {
+                "token_endpoint_auth_method": "private_key_jwt" //update to private_key_jwt
+            }
+        },
+        "settings": {
+            "oauthClient": {
+                "client_uri": "http://localhost:8080",
+                "logo_uri": "http://developer.okta.com/assets/images/logo-new.png",
+                 "redirect_uris": [
+                    "http://{yourOktadomain}/callback"   
+                ],
+                "response_types": [
+                    "token" //remove all options except token
+                ],
+                "grant_types": [
+                    "client_credentials" //replace all options with client_credentials   
+                ],
+                "application_type": "service", //change to service
+                "jwks": { //add the JWKS section with the values from your generated jwks
+                    "keys": [
+                        {
+                            "kty": "RSA",
+                            "e":"AQAB",
+                            "kid": "name your jwks",
+                            "n":"AJncrKuine49_CE…..RL0HU="
+                        }    
+                    ]
+                }
+            }
+        }
+    }
+    ```
+
+   You can also create a service client using the dynamic `/oauth/v1/clients` registration endpoint:
```

Comment:
for brevity, do we need a full request example here? Or could we just link to the /clients API doc?

Reply:
There isn't an example that includes a `jwks` on the /clients API doc page

Reply:
Then we could add it to the /clients page, since there will be developers who want to use private key JWT for reasons other than OAuth2 for Okta APIs.